ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml

```diff
@@ -0,0 +1,32 @@
+base_images:
+  tools:
+    name: "4.14"
+    namespace: ocp
+    tag: tools
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.14
+images:
+- dockerfile_path: prow/Dockerfile
```

Comment:
@jianzhangbjz 
```console
[root@417ccaa0b974 /]# which pip3
[root@417ccaa0b974 /]# which pip3
/usr/bin/pip3
[root@417ccaa0b974 /]# which pip
/usr/bin/which: no pip in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
[root@417ccaa0b974 /]# which python3
/usr/bin/python3

```

please use pip3 and python3

it will resolve https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/39619/rehearse-39619-periodic-ci-openshift-release-tests-master-stable-build-test/1661317142005420032/build-log.txt


so,
1, please add `#!/usr/bin/env python3` into https://github.com/openshift/release-tests/blob/master/prow/job/job.py
2, please change to `RUN pip3 install -e ./prow` in https://github.com/openshift/release-tests/blob/master/prow/Dockerfile

Reply:
Thanks! Mark this image `registry.ci.openshift.org/ocp/4.14:tools` here.